src/devhub/entity/community/Announcements.jsx

```diff
@@ -105,15 +119,18 @@ return (
                   },
                 },
               ]}
-              Item={(p) => (
+              Item={(item) => (
                 <Widget
-                  loading={
-                    <div className="w-100" style={{ height: "200px" }} />
-                  }
-                  src="${REPL_MOB}/widget/MainPage.N.Post"
+                  src="near/widget/Posts.Post"
```

Comment:
Since we want this to work in both mainnet and testnet, could you please use replacements?

${REPL_NEAR}

These replacements are applied on build during the dev script, you can find the value it is replaced with in replacements.<env>.json. This allows the code to work across regions